pkg/webui/console/views/device-general-settings/application-server-form/index.js

```diff
@@ -163,13 +177,17 @@ const ApplicationServerForm = React.memo(props => {
     >
       {showResetNotification && <Notification content={messages.keysResetWarning} info small />}
       <Form.Field
-        autoFocus
+        required
         title={sharedMessages.skipCryptoTitle}
-        name="skip_payload_crypto"
+        name="skip_payload_crypto_value"
         description={sharedMessages.skipCryptoDescription}
-        component={Checkbox}
+        component={RadioButton.Group}
```

Comment:
Have a look into the `encode`/ `decode` props of the `<Form.Field />`. You can use that to marshal values from the backend shape to the frontend shape and vice versa. Iirc the backend expects either `true`, `false` or `null`. If you use a `decode` function you can convert this into `skip`, `include` and `default` and the other way around.